pkg/apis/crd/v1alpha1/types.go

```diff
@@ -439,6 +439,12 @@ type NetworkPolicyPeer struct {
 	// Cannot be set with any other selector.
 	// +optional
 	ServiceAccount *NamespacedName `json:"serviceAccount,omitempty"`
+	// Select certain Nodes which match the label selector,
+	// if no nodeSelector is specified, then no additional Nodes IPs will be added
```

Comment:
I feel we can remove this sentence (line 443 - 444), as it does add useful information.

Reply:
Good catch, done.